pkg/cmd/audit/io.go

```diff
@@ -265,6 +266,95 @@ func PrintTopByVerbAuditEvents(writer io.Writer, numToDisplay int, events []*aud
 	}
 }
 
+func PrintLatencyTrackersStatsAuditEvents(writer io.Writer, events []*auditv1.Event) {
+	PrintSummary(writer, events)
+
+	latencyTrackers := map[string][]time.Duration{}
+	for _, event := range events {
+		for latencyTracker, latencyValue := range event.Annotations {
+			if !strings.HasPrefix(latencyTracker, "apiserver.latency.k8s.io/") {
+				continue
+			}
+
+			latencyDuration, err := time.ParseDuration(latencyValue)
+			if err != nil {
+				fmt.Println(fmt.Sprintf("Error parsing %q=%v duration, err=%v", latencyTracker, latencyValue, err))
+				continue
+			}
+			latencyTrackers[latencyTracker] = append(latencyTrackers[latencyTracker], latencyDuration)
+		}
+	}
+
+	for _, latencies := range latencyTrackers {
+		sort.Slice(latencies, func(i, j int) bool {
+			return latencies[i] < latencies[j]
+		})
+	}
+
+	type latencySummary struct {
+		min, max, median, p90 time.Duration
+	}
+	latencySummaries := map[string]latencySummary{}
+	for latencyTracker, latencies := range latencyTrackers {
+		min, max, median, p90 := statsForLatencyTrackers(90, latencies)
```

Comment:
as a follow up, can we make the percentile configurable?

Reply:
yes, this could be done :)